From 2040ec4b4dbc6d559c6ce6224148b96293e53470 Mon Sep 17 00:00:00 2001
From: Salvatore Bonaccorso <carnil@debian.org>
Date: Sat, 29 Sep 2012 12:14:54 +0200
Subject: [PATCH 2/2] Use dpkg-query --control-show interface

Use --control-show instead of --control-path which will dissapear at
some point[1].

 [1]: https://lists.debian.org/debian-devel/2012/09/msg00063.html
---
 policy.sgml |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index ba217e5..7c8f6a0 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -6360,7 +6360,7 @@ Built-Using: grub2 (= 1.99-9), loadlin (= 1.6e-1)
 		    normally found
 		    in <file>/var/lib/dpkg/info/*.symbols</file>, but
 		    packages should not rely on this and instead should
-		    use <tt>dpkg-query --control-path <var>package</var>
+		    use <tt>dpkg-query --control-show <var>package</var>
 		    symbols</tt> if for some reason these files need to be
 		    examined.
 		  </p>
@@ -6696,7 +6696,7 @@ Built-Using: grub2 (= 1.99-9), loadlin (= 1.6e-1)
 		    normally found
 		    in <file>/var/lib/dpkg/info/*.shlibs</file>, but
 		    packages should not rely on this and instead should
-		    use <tt>dpkg-query --control-path <var>package</var>
+		    use <tt>dpkg-query --control-show <var>package</var>
 		    shlibs</tt> if for some reason these files need to be
 		    examined.
 		  </p>
-- 
1.7.10.4

